lib/js/components/Buttons/IconButton/IconButton.vue

```diff
@@ -166,6 +173,12 @@
 		}
 	}
 
+	&.-loading {
+		.a-icon {
+			animation: a-fadeIn 300ms ease-in-out 0s 1, a-spinAround 1500ms infinite linear;
```

Comment:
I know we want to avoid styling child component, but here it's a content of slot and we are not touching Button's loading state. Button has 3 sizes, IconButton 5 and different sizes of icons. Thats why I didn't use Button state and added here second icon with condition state=loading.

Reply:
If it's acceptable, maybe I could move this definition of animation as some global variable.

Reply:
Did you try using `spinning` prop on `Icon`? Maybe we'll need to adjust spin time, but IMO it's valuable to unify default spinning time.

If it isn't possible, why don't you rotate based on `.a-iconButton__icon` selector?

Reply:
Thanks, I missed this `spinning` property. It works well.